<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mayo::DialogOptions</class>
 <widget class="QDialog" name="Mayo::DialogOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>231</width>
    <height>388</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Options</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox {
    font-weight: bold;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupBox_StlIo">
     <property name="title">
      <string>STL import/export</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioBtn_UseGmio">
        <property name="text">
         <string>Use gmio</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioBtn_UseOcc">
        <property name="text">
         <string>Use OpenCascade</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_BRepShapeGpx">
     <property name="title">
      <string>BRep shape defaults</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="toolBtn_BRepShapeDefaultColor">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Material</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_BRepShapeDefaultMaterial">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_MeshGpx">
     <property name="title">
      <string>Mesh defaults</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Color</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QToolButton" name="toolBtn_MeshDefaultColor">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Material</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="comboBox_MeshDefaultMaterial">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QWidget" name="widget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>1</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="3">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Show edges</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QCheckBox" name="checkBox_MeshShowEdges">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Show nodes</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBox_MeshShowNodes">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>1</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_ClipPlanes">
     <property name="title">
      <string>Clip planes</string>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QCheckBox" name="checkBox_Capping">
        <property name="text">
         <string>Capping</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QWidget" name="widget_CappingHatch" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Capping hatch</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox_CappingHatch"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>4</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>groupBox_StlIo</zorder>
  <zorder>groupBox_BRepShapeGpx</zorder>
  <zorder>groupBox_MeshGpx</zorder>
  <zorder>buttonBox</zorder>
  <zorder>verticalSpacer</zorder>
  <zorder>groupBox_ClipPlanes</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Mayo::DialogOptions</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Mayo::DialogOptions</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
